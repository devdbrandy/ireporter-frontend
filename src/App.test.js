import React from 'react';
import { shallow } from 'enzyme';
import Modal from 'react-modal';
import App from './App';

jest.mock('react-modal');

describe('<App />', () => {
  it('renders without crashing given the required props', () => {
    Modal.setAppElement.mockImplementation(() => null);
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
