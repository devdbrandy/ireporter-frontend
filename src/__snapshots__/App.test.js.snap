// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders without crashing given the required props 1`] = `<Routes />`;
